apps/financial-aid/web-osk/src/components/Summary/FormInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Box, Button, Divider, Icon, Text } from '@island.is/island-ui/core'
+import { useRouter } from 'next/router'
+
+interface InfoProps {
+  id: string
+  label: string
+  url: string
+  info?: string
+}
+
+interface Props {
+  info: InfoProps[]
+  error: boolean
+}
+
+const FormInfo = ({ info, error }: Props) => {
+  const router = useRouter()
+
+  return (
+    <>
+      {info.map((item, index) => {
+        const err = error && item.info === undefined
+        return (
+          <span key={'overview-' + index} id={item.id}>
+            <Divider />
+
+            <Box
+              display="flex"
+              justifyContent="spaceBetween"
+              alignItems="flexStart"
+              paddingY={[4, 4, 5]}
+            >
+              <Box marginRight={3}>
+                <Text fontWeight="semiBold" color={err ? 'red600' : 'dark400'}>
+                  {item.label} {err ? '*' : ''}
+                </Text>
+                <Text>{item.info}</Text>
```

Comment:
Since this can be undefined, should we do `?` or check if this is undefined before rendering it?